ethereum/PFP.sol

```diff
@@ -90,6 +98,8 @@ contract PFP is ERC721A, Ownable, VRFConsumerBase {
     }
 
     function setBaseURI(string memory baseURI_) external onlyOwner() {
+        require(!revealed, "Collection revealed, cannot set URI");
```

Comment:
I would strongly suggest this change (or one like it) that means the owner cannot change the URI after the reveal. This lets user's know that the metadata for their tokens cannot be changed after reveal.